modules/javafx.graphics/src/test/java/test/com/sun/javafx/scene/layout/region/BackgroundRepeatConverterTest.java

```diff
@@ -54,8 +53,8 @@
     /*
         -fx-background-repeat: null
      */
-    @Ignore ("this doesn't work, but I'm not sure what would happen with a null background-repeat in reality")
-    @Test public void scenario2() {
+    @Test(expected=NullPointerException.class)
```

Comment:
This is at odds with the body of the test, since presumably, the `assertEquals` will never be called.

An expect exception annotation is best used in the case where you do a single thing that is expected to generate that exception. Otherwise and unexpected exception could end up making the test look like it's passing.

Reply:
Hi Kevin, the test corrected now.
Removed `(expected=NullPointerException.class)` and added try, catch block.